### PR TITLE
Highlight cases with Sanger available for evaluation on cases page.

### PR DESCRIPTION
**Pull request by northwestwitch**
Showing a dismissible message with cases with Sanger available for evaluation at the top of the cases page.
This is to address #871

### CODE REVIEW

**Comment by moonso** [comment]
Hi @northwestwitch , what is the status on this one? Ready to merge?

**Comment by northwestwitch** [comment]
If we want to have it yes, I guess..

File: scout/server/blueprints/cases/controllers.py [hunk 1]
```diff
@@ -203,6 +203,42 @@ def rerun(store, mail, current_user, institute_id, case_name, sender, recipient)
     mail.send(msg)
 
 
+def get_sanger_unevaluated(store, institute_id):
+    """Get all variants for an institute having Sanger validations ordered but still not evaluated"""
```
Comment (moonso) [inline scout/server/blueprints/cases/controllers.py]: Could you complete this docstring?

**Comment by northwestwitch** [comment]
Creating a new one, it's easier since this one is oldish and has too many conflicts to solve.

**Pull request closed without merging**